update.php

```diff
@@ -1061,3 +1053,20 @@ function update_1442()
 
 	return Update::SUCCESS;
 }
+
+/**
+ * A bug in Contact\User::updateByContactUpdate prevented any update to the user-contact table since the rows have been
+ * created in version 1435. This version fixes this bug but the user-contact rows are outdated, we need to regenerate
+ * them.
+ */
+function update_1444()
+{
+	DBA::e('TRUNCATE TABLE `user-contact`');
```

Comment:
Why are you deleting the contact table here?

Reply:
We haven't been updating the `user-contact` rows since update 1435 because of a bug in `Contact\User::updateByContactUpdate`, so we need to start fresh to be able to use the table in the new modules.

Reply:
Which bug?

Reply:
A misnamed variable made the condition in `Contact\User::updateByContactUpdate` always fail which prevented the update on the `user-contact` table.

See https://github.com/friendica/friendica/pull/10969/commits/888d1492d3af0190bea5c8a1db574fa042ee704b